StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetIntentConfiguration.swift

```diff
@@ -34,27 +34,6 @@ public extension PaymentSheet {
             _ intentCreationCallback: @escaping ((Result<String, Error>) -> Void)
         ) -> Void
 
-        /// Creates a `PaymentSheet.IntentConfiguration` with the given values
-        /// - Parameters:
-        ///   - mode: The mode of this intent, either payment or setup
-        ///   - paymentMethodTypes: The payment method types for the intent
-        ///   - onBehalfOf: The account (if any) for which the funds of the intent are intended
-        ///   - paymentMethodConfigurationId: Configuration ID (if any) for the selected payment method configuration
-        ///   - confirmHandler: A handler called with payment details when the user taps the primary button (e.g. the "Pay" or "Continue" button).
-        public init(mode: Mode,
```

Comment:
Might be missing something, but how is this not a breaking change?

Reply:
I guess the new init with `requireCVCRecollection` which has a default value takes it place?

Reply:
Yea, the diff is weird looking. It should not be a breaking change.